src/User32.Desktop/User32.cs

```diff
@@ -57,7 +56,61 @@ public static partial class User32
+        /// <returns>
+        /// An LRESULT. Usually if nCode is less than zero, the hook procedure must return the value returned by CallNextHookEx.
+        /// If nCode is greater than or equal to zero, it is highly recommended that you call CallNextHookEx and return the value it returns;
+        /// otherwise, other applications that have installed hooks will not receive hook notifications and may behave incorrectly as a result.
+        /// If the hook procedure does not call CallNextHookEx, the return value should be zero.
+        /// </returns>
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        public delegate int WindowsHookDelegate(int nCode, IntPtr wParam, IntPtr lParam);
+
+        /// <summary>
+        /// An application-defined callback function used with the <see cref="EnumWindowStations(WINSTAENUMPROC, IntPtr)"/> function.
+        /// </summary>
+        /// <param name="lpszWindowStation">The name of the window station.</param>
+        /// <param name="lParam">An application-defined value specified in the <see cref="EnumWindowStations(WINSTAENUMPROC, IntPtr)"/> function.</param>
+        /// <returns>To continue enumeration, the callback function must return TRUE (non-zero value). To stop enumeration, it must return FALSE (0).</returns>
+        /// <remarks>
+        /// An application must register this callback function by passing its address to <see cref="EnumWindowStations(WINSTAENUMPROC, IntPtr)"/>.
+        /// The callback function can call SetLastError to set an error code for the caller to retrieve by calling GetLastError.
+        /// </remarks>
+        [UnmanagedFunctionPointer(CallingConvention.StdCall, CharSet = CharSet.Unicode)]
+        public delegate int WINSTAENUMPROC(string lpszWindowStation, IntPtr lParam);
+
+        /// <summary>
+        /// An application-defined callback function used with the <see cref="EnumDesktops(SafeWindowStationHandle, DESKTOPENUMPROC, IntPtr)"/> function.
+        /// </summary>
+        /// <param name="lpwstrDesktopName">The name of the desktop.</param>
+        /// <param name="lParam">An application-defined value specified in the <see cref="EnumDesktops(SafeWindowStationHandle, DESKTOPENUMPROC, IntPtr)"/> function.</param>
+        /// <returns>To continue enumeration, the callback function must return TRUE (non-zero value). To stop enumeration, it must return FALSE (0).</returns>
+        /// <remarks>
+        /// An application must register this callback function by passing its address to <see cref="EnumDesktops(SafeWindowStationHandle, DESKTOPENUMPROC, IntPtr)"/>.
+        /// The callback function can call SetLastError to set an error code for the caller to retrieve by calling GetLastError.
+        /// </remarks>
+        [UnmanagedFunctionPointer(CallingConvention.StdCall, CharSet = CharSet.Unicode)]
+        public delegate int DESKTOPENUMPROC(string lpwstrDesktopName, IntPtr lParam);
```

Comment:
See the [comment above](https://github.com/AArnott/pinvoke/pull/268#discussion_r74706181)